data/dashboard.yml

```
@@ -41,6 +41,17 @@
         - name: Repositories
           url: /repos.html
           description: All the applications and other repositories we use to build GOV.UK.
+    - name: GOV.UK documentation
+      sites:
+        - name: 2nd-line Support Manuals
+          url: /manual/2nd-line.html
+          description: The documentation for GOV.UK's "Second-line Support"
+        - name: Developer Documentation
+          url: /manual.html
+          description: The contents page of GOV.UK's Developer Documentation
+        - name: Kubernetes Documentation
```

nit: slightly better to clarify that it's GOV.UK-specific as opposed to just _the_ Kubernetes documentation?
```suggestion
        - name: GOV.UK Kubernetes documentation
```